recipes/libxml2/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export CPPFLAGS="-I${PREFIX}/include"
+
+if [ $(uname) == Darwin ]; then
+  export OPTS="--disable-dependency-tracking --without-lzma"
+else
+  export OPTS="--with-lzma=$PREFIX"
```

Comment:
I missed the history here. Why are we doing this on some platforms and not others?